HTAN_schema.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+
+import pandas as pd
+
+from schema_explorer import SchemaExplorer
```

Comment:
Update this with new path to `SchemaExplorer`.

Reply:
Yes - that one slipped under the radar... will update.

Reply:
Where do you think files like HTAN_schema.py should live?
I don't want them to hang around in the main directory. 

These are in a way examples of how to use the csv_2_schema script to create schemas, so they are sort of on the border between schema utils and schema creation examples?

Reply:
Since `HTAN_schema.py` is essentially an example to test the `csv_2_schemaorg.py` module perhaps we can follow a similar structure as we did with other sub-packages and rename `HTAN_schema.py` to `csv_to_schemaorg_usage` and put it into an `examples` sub-folder?

Like this:
ingresspipe
- config
- manifest
- models
- schemas
- utils
    * other utils *
    - csv_utils
    - examples
       - csv_to_schemaorg_usage

Reply:
I like this idea - could you handle that one?

Reply:
Sure! Will do.